client/src/unifycr-fixed.c

```diff
@@ -363,6 +364,7 @@ static int unifycr_logio_chunk_write(
         ssize_t rc = __real_pwrite(unifycr_spilloverblock, buf, count, spill_offset);
         if (rc < 0)  {
             perror("pwrite failed");
+            LOGERR("pwrite failed");
```

Comment:
Let's also add the errno/strerror(errno) info that we'd get from perror(), e.g,.:

LOGERR("pwrite failed: errno=%d %s", errno, strerror(errno))

Reply:
That's a good idea.  I'm good with that.

Reply:
Latest pushed version has these changes. Ready for review/merge.